flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/runtime/stream/sql/AggregateITCase.scala

```diff
@@ -1334,4 +1334,32 @@ class AggregateITCase(
     val expected = Seq("3,29.39,tom123@gmail.com")
     assertEquals(expected.sorted, sink.getRetractResults.sorted)
   }
+
+  @Test
+  def testSplitAggsHandler(): Unit = {
```

Comment:
Could you rename to `testAggregationCodeSplit` ?  We have a `SplitAggregateITCase` test which may be confuse users the naming.